packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.ts

```diff
@@ -3,9 +3,38 @@ import { Component } from '@angular/core';
 @Component({
   selector: '<%= prefix %>-root',<% if (inlineTemplate) { %>
   template: `
-    <h1>
-      {{title}}
-    </h1><% if (routing) { %>
+    <!--The content below can be removed with the new code.-->
+    <div style="text-align:center">
+      <h1>
+        Welcome to {{title}}!!
+      </h1>
+      <img width="300" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4N
```

Comment:
Isn't it better to save the file in the `assets` folder and reference it here? Would make the file a lot cleaner.